src/open_inwoner/openzaak/tests/test_notification_zaak_status.py

```diff
@@ -569,24 +569,34 @@ def test_handle_status_update(self, mock_send: Mock, mock_feed_hook: Mock):
         case = factory(Zaak, data.zaak)
         case.zaaktype = factory(ZaakType, data.zaak_type)
 
-        status = factory(Status, data.status_final)
-        status.statustype = factory(StatusType, data.status_type_final)
+        status1 = factory(Status, data.status_initial)
+        status1.statustype = factory(StatusType, data.status_type_initial)
+
+        status2 = factory(Status, data.status_final)
+        status2.statustype = factory(StatusType, data.status_type_final)
```

Comment:
A general feedback: let's stop arbitrary numbering our test data and use meaningful postfixes so things stay readable. 

In this case you don't even have to invent a scheme but can just copy form the data class, eg: `status1` is `status_initial` and `status2` is `status_final` (as well as the related objects).